pkgs/by-name/ls/lshw/package.nix

```diff
@@ -11,24 +11,22 @@
 
 stdenv.mkDerivation rec {
   pname = "lshw";
-  # FIXME: when switching to a stable release:
   # Fix repology.org by not including the prefixed B, otherwise the `pname` attr
   # gets filled as `lshw-B.XX.XX` in `nix-env --query --available --attr nixpkgs.lshw --meta`
   # See https://github.com/NixOS/nix/pull/4463 for a definitive fix
-  version = "unstable-2023-03-20";
+  version = "02.20";
 
   src = fetchFromGitHub {
     owner = "lyonel";
     repo = pname;
```

Comment:
https://github.com/NixOS/nixpkgs/pull/278611

Reply:
From the [tracking issue](https://github.com/NixOS/nixpkgs/issues/277994) it seems usage of `pname` in `rec` in fine, so I will ignore this issue for now until we have a conclusion for that issue.

Reply:
BTW, I will eventually rewrite my derivations to use `finalAttrs`. But today is not this day.